app/models/proposal/token.rb

```diff
@@ -62,7 +62,7 @@ def validate_accepted
     scope :reminded, ->{ where('reminded_at IS NOT NULL') }
 
     def expects
-      @expects || proposable.proposal_options[:expects]
+      @expects ||= proposable.proposal_options[:expects]
```

Comment:
I don't see why we would need todo this change, it's not an expensive operation so no point adding?